generic_config_updater/patch_sorter.py

```diff
@@ -36,6 +36,13 @@ def apply_move(self, move):
     def has_no_diff(self):
         return self.current_config == self.target_config
 
+    def __str__(self):
```

Comment:
Adding `__str__` and `__repr__` are not related to this PR, but they are small improvements to the `Diff` class.